README.md

```diff
@@ -1,40 +1,79 @@
 # Signon
 
-Signon is a centralised OAuth2 based single sign-on provider for GDS services that provides username/password and 2-Factor authentication.
+Signon is a centralised, OAuth2-based single sign-on (SSO) provider for GDS services that provides username/password and 2-factor authentication.
 
-## Technical documentation
```

Comment:
This is part of the convention in: https://docs.publishing.service.gov.uk/manual/readmes.html your point is right that this is all technical documentation to a degree but I don't think this is worth breaking the convention for.